gatekeeper/src/test/java/eu/arrowhead/relay/gatekeeper/activemq/ActiveMQGatekeeperRelayClientTest.java

```diff
@@ -114,35 +113,35 @@ public void setUp() throws KeyStoreException, NoSuchAlgorithmException, Certific
 		otherPrivateKey = Utilities.getPrivateKey(keystore2, "123456");
 	}
 	
-	//-------------------------------------------------------------------------------------------------
-	@Test(expected = IllegalArgumentException.class)
-	public void testConstructorServerCommonNameNull() {
-		GatekeeperRelayClientFactory.createGatekeeperRelayClient(null, null, null, null, 0);
-	}
-
-	//-------------------------------------------------------------------------------------------------
-	@Test(expected = IllegalArgumentException.class)
-	public void testConstructorServerCommonNameEmpty() {
-		GatekeeperRelayClientFactory.createGatekeeperRelayClient(" ", null, null, null, 0);
-	}
-	
-	//-------------------------------------------------------------------------------------------------
-	@Test(expected = IllegalArgumentException.class)
-	public void testConstructorPublicKeyNull() {
-		GatekeeperRelayClientFactory.createGatekeeperRelayClient("gatekeeper.testcloud2.aitia.arrowhead.eu", null, null, null, 0);
-	}
-	
-	//-------------------------------------------------------------------------------------------------
-	@Test(expected = IllegalArgumentException.class)
-	public void testConstructorPrivateKeyNull() {
-		GatekeeperRelayClientFactory.createGatekeeperRelayClient("gatekeeper.testcloud2.aitia.arrowhead.eu", clientPublicKey, null, null, 0);
-	}
-	
-	//-------------------------------------------------------------------------------------------------
-	@Test(expected = IllegalArgumentException.class)
-	public void testConstructorSSLPropertiesNull() {
-		GatekeeperRelayClientFactory.createGatekeeperRelayClient("gatekeeper.testcloud2.aitia.arrowhead.eu", clientPublicKey, otherPrivateKey, null, 0);
-	}
+//	//-------------------------------------------------------------------------------------------------
```

Comment:
Please remove these commented tests if we not use them.

Reply:
Don't understand why, but seems GitHUB doesn't load the latest changes on this class. I remeber that I have fixed these, locally I see that they are fixed, but here in GitHUB I see the `Changed since last view` label, but cannot reload the file...

Reply:
See the file itself in GitHUB on the gateway-upgrade branch is also correct...
https://github.com/arrowhead-f/core-java-spring/blob/gateway-upgrade/relay-library/src/test/java/eu/arrowhead/relay/gatekeeper/activemq/ActiveMQGatekeeperRelayClientTest.java

Reply:
Ah there are two ActiveMQGatekeeperRelayClientTest in different packages.
Fixed